### PR TITLE
Few more fixes for issues found during tests

### PR DESCRIPTION
**Pull request by wutschel**
Several fixes for issues reported from tester or found by myself:

- Fix Dark Mode for few items (TV pie chart, SettingsValuesVC item and MoreItemsVC item)
- Hide button5 (...) when unsupported. Avoids crash on iPad and unwanted highlighting when pressing empty area.
- Instead of searchbar provide an info label on top of the collectionView ([forum post](https://forum.kodi.tv/showthread.php?tid=359717&pid=3009849#pid3009849))

### CODE REVIEW

File: XBMC Remote/iPad/MenuViewController.m [hunk 1]
```diff
@@ -54,7 +54,7 @@ @implementation MenuViewController
 - (id)initWithFrame:(CGRect)frame mainMenu:(NSMutableArray *)menu{
     if (self = [super init]) {
 		[self.view setFrame:frame]; 
-        int tableHeight = ([menu count] -1) * PAD_MENU_HEIGHT + PAD_MENU_INFO_HEIGHT;
+        CGFloat tableHeight = ([menu count] -1) * PAD_MENU_HEIGHT + PAD_MENU_INFO_HEIGHT;
```
Comment (kambala-decapitator) [inline XBMC Remote/iPad/MenuViewController.m:57]: while you're here, please:
1. fix spacing inside parantheses
2. fix indentation of adjacent lines

File: XBMC Remote/DetailViewController.m [hunk 2]
```diff
@@ -5703,11 +5704,21 @@ - (void)viewDidLoad{
     else {
         self.searchController.searchBar.hidden = NO;
     }
-    frame = collectionView.pullToRefreshView.frame;
-    frame.origin.y = 0;
-    collectionView.pullToRefreshView.frame = frame;
     activeLayoutView = dataList;
     
+    // For CollectionView place an info label at the position of the searchbar
+    UIView *infobar = [[UIView alloc] initWithFrame:CGRectMake(0, 0, viewWidth, self.searchController.searchBar.frame.size.height)];
```
Comment (kambala-decapitator) [inline XBMC Remote/DetailViewController.m:5710]: does it resize correctly on rotation? Don't see `autoresizingMask`.

File: XBMC Remote/DetailViewController.m [hunk 3]
```diff
@@ -5703,11 +5704,21 @@ - (void)viewDidLoad{
...
+    infobar.backgroundColor = [UIColor clearColor];
+    UILabel *infolabel = [[UILabel alloc] initWithFrame:CGRectMake(INFO_PADDING, INFO_PADDING, viewWidth - 2*INFO_PADDING, self.searchController.searchBar.frame.size.height - 2*INFO_PADDING)];
+    infolabel.backgroundColor = collectionViewSearchBarColor;
+    infolabel.textColor = [UIColor darkGrayColor];
+    infolabel.text = [NSString stringWithFormat:@" %@", NSLocalizedString(@"For search switch to list view", nil)];
```
Comment (kambala-decapitator) [inline XBMC Remote/DetailViewController.m:5715]: 1. what's the point of the leading space?
2. the string isn't added to English strings

**Review by kambala-decapitator — CHANGES_REQUESTED** [review]

File: XBMC Remote/iPad/MenuViewController.m (hunk 1 above)
Comment (wutschel) [reply XBMC Remote/iPad/MenuViewController.m:57]: Will be done with next forced push.

File: XBMC Remote/DetailViewController.m (hunk 3 above)
Comment (wutschel) [reply XBMC Remote/DetailViewController.m:5715]: 1. Without leading space the text is too far left. It needs some padding.
2. Will be done with next forced push.

File: XBMC Remote/DetailViewController.m (hunk 2 above)
Comment (wutschel) [reply XBMC Remote/DetailViewController.m:5710]: It works. The width of the stacked views is fixed and doesn´t change when rotating. Of course I can add

```
infobar.autoresizingMask = UIViewAutoresizingFlexibleWidth;
infolabel.autoresizingMask = UIViewAutoresizingFlexibleWidth;
```

Comment (kambala-decapitator) [reply XBMC Remote/DetailViewController.m:5710]: thanks, no need then

**Comment by wutschel** [comment]
Updated as discussed.

**Pull request merged**